bundles/pubsub/pubsub_admin_zmq/src/psa_activator.c

```diff
@@ -62,7 +62,7 @@ int psa_zmq_start(psa_zmq_activator_t *act, celix_bundle_context_t *ctx) {
     //track serializers
     if (status == CELIX_SUCCESS) {
         celix_service_tracking_options_t opts = CELIX_EMPTY_SERVICE_TRACKING_OPTIONS;
-        opts.filter.serviceName = PUBSUB_SERIALIZER_SERVICE_NAME;
+        opts.filter.serviceName = PUBSUB_MESSAGE_SERIALIZATION_SERVICE_NAME;
```

Comment:
The marker service can be used here so service name
PUBSUB_MESSAGE_SERIALIZATION_MARKER_NAME

note that the pubsub_serializer_handler_t can be used to actual use the services (and reuse the tracking of services)

Reply:
I think the marker interfaces complicate things more than what they intend to do. Performance-wise, I don't see any issues with iterating over a couple hundred serializers during setup/teardown. But let's have a call on this.

Reply:
The marker interface was not intended for performance issues, but intended to prevent no available serialiser issues.

So if json serializing is supported, but at the moment no actual serialiser is available, the marker interface is present to indicate json serialization is supported.

But maybe this is over designed ... so yeah let's discuss this.